layouts/shortcodes/observability_pipelines/processors/generate_metrics.en.md

```diff
@@ -11,9 +11,10 @@ Click **Manage Metrics** to create new metrics or edit existing metrics. This op
 
  1. Enter a [filter query](#filter-query-syntax). Only logs that match the specified filter query are processed. All logs, regardless of whether they match the filter query, are sent to the next step in the pipeline. **Note**: Since a single processor can generate multiple metrics, you can define a different filter query for each metric.
 1. Enter a name for the metric.
-1. In the **Define parameters** section, select the metric type (count, gauge, or distribution). See [Metrics Types](#metrics-types) for more information.
+1. In the **Define parameters** section, select the metric type (count, gauge, or distribution). See the [Count metric example]() and [Distribution metric example](). Also see [Metrics Types](#metrics-types) for more information.
     - For gauge and distribution metric types, select a log field which has a numeric (or parseable numeric string) value that is used for the value of the generated metric.
     - For the distribution metric type, the log field's value can be an array of (parseable) numerics, which is used for the generated metric's sample set.
+    - The **Group by** field determines how the metric values are grouped together. For example, if you have hundreds of hosts spread across four regions, grouping by region allows you to graph one line for every region.
```

Comment:
This is great but I think it could be useful maybe to call out that specifically that the fields listed in the group by setting are set as tags on the configured metric.

Reply:
```suggestion
    - The **Group by** field determines how the metric values are grouped together. For example, if you have hundreds of hosts spread across four regions, grouping by region allows you to graph one line for every region. The fields listed in the **Group by** setting are set as tags on the configured metric.
```

Reply:
Thanks @neuronull! I added it as a suggestion. When you say "configured metric", is that the metric that is generated by the processor?